openedx/features/course_experience/static/course_experience/fixtures/course-home-fragment.html

```diff
@@ -110,6 +86,21 @@ <h3 class="hd-6 section-title">Course Tools</h3>
                         </li>
                     </ul>
                 </div>
+                <div class="section section-upgrade">
+                    <h3 class="hd hd-6">Pursue a verified certificate</h3>
+                        <div class="upgrade-container">
+                            <p>
+                                <a class="btn-brand btn-upgrade"
+                                   href="${upgrade_url}"
+                                   data-creative="sidebarupsell"
```

Comment:
thoughts on naming?

Reply:
I used `sidebar_hero` in the original test if that sounds more appealing.

Reply:
well the ticket says 
https://openedx.atlassian.net/browse/LEARNER-3058
"Update location / position label to "sidebar-message" (or something similar) from "hero" and "sidebar-hero" (which was used in the test)"
so i'm guessing marco wanted to be able to differentiate from the test version, though he said he had no suggestions for this :man_shrugging: